src/app/rundown/components/part/part.component.ts

```diff
@@ -23,20 +27,70 @@ import {animate, state, style, transition, trigger} from '@angular/animations';
     ]),
   ]
 })
-export class PartComponent {
+export class PartComponent implements OnChanges {
 
   @Input()
   public isRundownActive: boolean
 
+  // TODO: Figure out how to "subscribe" to changes for this on the part property.
+  @Input()
+  public isOnAir: boolean
+
+  @Input()
+  public time: number
+
   @Input()
   public part: Part
 
+  @Input()
+  public pieceLayers: string[]
+
+  @Input()
+  public pixelsPerSecond: number
```

Comment:
This is starting to become a lot of inputs for a component.

Same applies for some of the other components

Reply:
Yes.